selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/SeleniumWebDriver.java

```diff
@@ -255,7 +255,8 @@ private RemoteWebDriver doCreateDriver(URL webDriverUrl) {
     switch (browser) {
       case GOOGLE_CHROME:
         LoggingPreferences loggingPreferences = new LoggingPreferences();
-        loggingPreferences.enable(LogType.BROWSER, Level.SEVERE);
+        loggingPreferences.enable(LogType.PERFORMANCE, Level.ALL);
```

Comment:
There was Level.SEVERE used for some reasons. 
@Ohrimenko1988: please, comment on this.

Reply:
It was used because only browser errors have useful information for us. But it only in case for browser's console output. In other cases (for example "Networking"), logs of other levels may be useful.